client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -23,7 +23,6 @@ import { ViewChangeReco } from '../../models/view-change-reco';
 import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
 import { ViewUnifiedChange } from '../../models/view-unified-change';
 import { OperatorService } from '../../../../core/services/operator.service';
-import { CategoryRepositoryService } from '../../services/category-repository.service';
 
 /**
  * Component for the motion detail view
```

Comment:
parameters in documentation are outdated

Reply:
Done in another PR.